client/model/key.go

```diff
@@ -45,6 +46,30 @@ const (
 //   {B: [flat, flat], G: [sharp], E: [flat]}
 type KeySignature map[NoteLetter][]Accidental
 
+func (ks KeySignature) String() string {
```

Comment:
@daveyarwood I added this pretty hastily. Let me know if something like it exists / it can be done more easily / we rename the function.

Reply:
This looks reasonable to me, and I don't think we have anything like it yet :+1: